### PR TITLE
Sphinx doc chapter 9

### PR DESCRIPTION
**Pull request by maximedenes**

### CODE REVIEW

File: doc/sphinx/proof-engine/ltac.rst [hunk 1]
```diff
@@ -1,1829 +1,1256 @@
...
+   every time. It is first-order except on the variables of the form :n:`@?id`
+   that occur in head position of an application. For these variables, the
+   matching is second-order and returns a functional term.
+
+   Alternatively, when a metavariable of the form :n:`?id` occurs under binders,
+   say :n:`x__1, …, x__n` and the expression matches, the
+   metavariable is instantiated by a term which can then be used in any
+   context which also binds the variables :n:`x__1, …, x__n` with
+   same types. This provides with a primitive form of matching under
+   cont
```
Comment (Zimmi48) [inline doc/sphinx/proof-engine/ltac.rst]: There is a quoting problem here.

File: doc/sphinx/proof-engine/ltac.rst [hunk 2]
```diff
@@ -1,1829 +1,1256 @@
...
-Tactic expressions can only be applied in the context of a proof.  The
+.. include:: ../preamble.rst
+.. include:: ../replaces.rst
+
+.. _ltac:
+
+The tactic language
+===================
+
+This chapter gives a compact documentation of Ltac, the tactic language
```
Comment (Zimmi48) [inline doc/sphinx/proof-engine/ltac.rst]: Could use |Ltac| here.

File: doc/sphinx/proof-engine/ltac.rst [hunk 3]
```diff
@@ -1,1829 +1,1256 @@
...
-%% \caption{Values of ${\cal L}_{tac}$}
-%% \label{ltacval}
-%% \end{figure}
-
-%% \subsection{Evaluation}
-
-\subsubsection[Sequence]{Sequence\tacindex{;}
-\index{Tacticals!;@{\tt {\tac$_1$};\tac$_2$}}}
+The rest of this section explains the semantics of every construction of
+Ltac.
```
Comment (Zimmi48) [inline doc/sphinx/proof-engine/ltac.rst]: Same remark.

File: doc/sphinx/proof-engine/ltac.rst [hunk 4]
```diff
@@ -1,1829 +1,1256 @@
...
+   tries to apply :n:`v__2` and so on. It fails when there is no
+   applicable tactic. In other words,
+   :n:`first [:@expr__1 | ... | @expr__n]` behaves, in each goal, as the the first
+   :n:`v__i` to have *at least* one success.
+
+   .. exn:: Error message: No applicable tactic
+
+   .. tacv:: first @expr
+
+      This is an Ltac alias that gives a primitive access to the first
```
Comment (Zimmi48) [inline doc/sphinx/proof-engine/ltac.rst]: Same remark.

File: doc/sphinx/replaces.rst [hunk 5]
```diff
@@ -36,6 +36,7 @@
 .. |ident_n,k_n| replace:: `ident`\ :math:`_{n,k_n}`
 .. |ident_n| replace:: `ident`\ :math:`_{n}`
 .. |L_tac| replace:: `L`:sub:`tac`
+.. |Ltac| replace:: `L`:sub:`tac`
```
Comment (Zimmi48) [inline doc/sphinx/replaces.rst]: The macro above should eventually be removed in favor of this one.

**Review by Zimmi48 — COMMENTED** [review]
Impressive work :+1:

File: doc/sphinx/replaces.rst (hunk 5 above)
Comment (maximedenes) [reply doc/sphinx/replaces.rst]: Yes, that's the plan.

**Comment by maximedenes** [comment]
Comments addressed, rebased.

**Comment by Zimmi48** [comment]
CI failures were expected and are unrelated.

**Pull request merged**